### DISCUSSION TITLE
Template literals on API rules

### DISCUSSION THREAD

**Discussion by jesusmdy** [pocketbase/pocketbase#362] [Q&A] [answered]
Does Pocketbase API rules suppport template literals as method to verify strings?

I want to test a string that comes from the client and contains a join of the postId and the userId to form an extra identifier for a likes collection.

![image](https://user-images.githubusercontent.com/26631510/187477263-7c9d1682-400c-44fd-b789-edf5e506d57c.png)

**Comment by ganigeorgiev** [comment, marked as answer]
No, this is not supported. You can't use template literals or inline expressions in the rules/filters syntax.

If you want to do more advanced create/update data validations, you'll have to:
- use PocketBase as framework with the [create/update event hooks](https://pocketbase.io/docs/event-hooks/)
- OR use a middleware service between the client and PocketBase (suitable if you are already using SSR for example).
